developers.adoc

```diff
@@ -175,6 +194,55 @@ actually be JavaScript libraries, not Java jars.
 . Go to https://oss.sonatype.org/#stagingRepositories
 . Continue to follow Java runtime publishing instructions
 
+== Developing new feature
+
+Usually, you will develop new feature in your own fork of sub-project and then
```

Comment:
Just give a link to https://guides.github.com/introduction/flow/

Reply:
Not everyone will spend time reading what they need just to share their little solution with the world. Therefore, there should be specific instructions here. Link also can be provided, but for curious

Reply:
>Not everyone will spend time reading

These are obvious things. You cannot contribute (you can fork and do what you want in an own fork, but on the first PR sent without following the flow a PR author will be asked either to follow the flow or GTFO) on GitHub (and GitLab, and BitBucket) without knowing them. We can expect everyone on GitHub already knowing the flow.

Reply:
>  We can expect everyone on GitHub already knowing the flow.

Why then reference to https://guides.github.com/introduction/flow/ needed? That sentence here just for glue and to justify the need for actions below.

Reply:
Would be fine to remove that part completely.